plugins/BEdita/Core/src/Model/Behavior/GeometryBehavior.php

```diff
@@ -100,10 +101,12 @@ public static function parseCoordinates($point)
             $point = preg_split('/[\s,]/', (string)$point, 2);
         }
         $point = filter_var_array(array_values($point), FILTER_VALIDATE_FLOAT);
-        if (count($point) !== 2 || in_array(false, $point, true) || abs($point[0]) > 90 || abs($point[1]) > 180) {
+
+        $valid = LocationsValidator::checkCoordinates($point);
```

Comment:
minor advice: rename `$valid` to something that can represents both boolean and string result. I.e. `$result`